product_docs/docs/pgd/5/manual/07-configure-proxies.mdx

```diff
@@ -0,0 +1,255 @@
+properties:
+
+The name of the PGD cluster's top-level group (as `name`).
+An array of endpoints of databases (as `endpoints`).
+The proxy definition object with a name and endpoint (as `proxy`).
+
+The first two properties will be the same for all hosts:
+
+```
+cluster:
+  name: pgd
+  endpoints:
+    - host=host-one dbname=bdrdb port=5444
+    - host=host-two dbname=bdrdb port=5444
+    - host=host-three dbname=bdrdb port=5444
+```
+
+Note that the endpoints in this example specify port=5444.
+This is necessary for EDB Postgres Advanced Server instances.
+For EDB Postgres Extended and Community PostgreSQL, this can be omitted.
+
+The third property, `proxy`, has a `name` property and an `endpoint` property.
+The `name` property should be a name created with `bdr.create_proxy` earlier, and it will be different on each host.
+The `endpoint` property is a string which defines how the proxy presents itself as a connection string.
+A proxy cannot be on the same port as the Postgres server and, ideally, should be on a commonly used port different from direct connections, even when no Postgres server is running on the host.
+We typically use port 6432 for PGD proxies.
+
+```
+  proxy:
+    name: pgd-proxy-one
+    endpoint: "host=localhost dbname=bdrdb port=6432"
+```
+
+## Install a PGD proxy configuration on each host
+
+For each host, create the `/etc/edb/pgd-proxy` directory:
+
+```
+sudo mkdir -p /etc/edb/pgd-proxy
+```
+
+Then on each host, write the appropriate configuration to the `pgd-proxy-config.yml` file in the `/etc/edb/pgd-proxy` directory.
+
+For our example, this could be run on host-one to create the file.
+
+```
+cat <<EOF | sudo tee /etc/edb/pgd-proxy/pgd-proxy-config.yml
+cluster:
+  name: pgd
+  endpoints:
+    - host=host-one dbname=bdrdb port=5444
+    - host=host-two dbname=bdrdb port=5444
+    - host=host-three dbname=bdrdb port=5444
+  proxy:
+    name: pgd-proxy-one
+    endpoint: "host=localhost dbname=bdrdb port=6432"
+EOF
+```
+
+## Restart the service
+
+```
+sudo systemctl restart pgd-proxy
+```
+
+Confirm it is running correctly with:
+
+```
+sudo systemctl status pgd-proxy
+```
+
+It should show `Active: (running)` in the opening details.
+
+
+## Test the proxy
+
+At this point, connecting to the PGD Proxy port on any node in the cluster will result in the connection being routed to the current write lead node. 
+
+For example, and assuming you've installed the proxy on all three hosts, then connecting to the proxy on host-three should result in the connection being routed to node-one:
+
+Here we pass connection details, using the `-d` flag of `psql` with the hostname for the proxy we just configured and the proxy port number:
+
+```shell
+sudo -iu enterprisedb psql -d "host=host-three dbname=bdrdb port=6432"
+__OUTPUT__
+psql (16.1.0, server 16.1.0)
+Type "help" for help.
+
+bdrdb=#
+```
+
+Once connected, we can query the server to find out which node the proxy has connected us to.
```

Comment:
This was a bit awkward, since we know the proxy should have connected us to the current write leader.   Maybe first have them query which node is write leader, then verify that's the node the proxy connected us to?